schema-lending.graphql

```diff
@@ -570,19 +609,48 @@ type Market @entity {
   " Sum of all historical liquidations in USD "
   cumulativeLiquidateUSD: BigDecimal!
 
+  " Sum of all historical transfers in USD "
+  cumulativeTransferUSD: BigDecimal!
+
+  " Sum of all historical flashloans in USD "
+  cumulativeFlashloanUSD: BigDecimal!
+
+  ##### Usage Data #####
+
+  " Number of cumulative unique users "
+  cumulativeUniqueUsers: Int!
+
+  " Number of cumulative depositors "
+  cumulativeUniqueDepositors: Int!
+
+  " Number of cumulative borrowers "
+  cumulativeUniqueBorrowers: Int!
+
+  " Number of cumulative liquidators (accounts that performed liquidation) "
+  cumulativeUniqueLiquidators: Int!
+
+  " Number of cumulative liquidatees (accounts that got liquidated) "
+  cumulativeUniqueLiquidatees: Int!
+
   ##### Token Balances #####
 
-  " Amount of input tokens in the market "
-  inputTokenBalance: BigInt!
+  " Amount of input tokens in the market (same order as inputTokens) "
+  inputTokensBalance: [BigInt!]!
 
   " Price per share of input token in USD "
   inputTokenPriceUSD: BigDecimal!
 
+  " Amount of input tokens borrowed in this market using variable interest rates "
+  variableBorrowedTokenBalance: BigInt
+
+  " Amount of input tokens borrowed in this market using stable interest rates "
+  stableBorrowedTokenBalance: BigInt
+
   " Total supply of output token "
-  outputTokenSupply: BigInt!
+  outputTokenSupply: BigInt
 
   " Price per share of output token in USD "
-  outputTokenPriceUSD: BigDecimal!
+  outputTokenPriceUSD: BigDecimal
 
   " Amount of input token per full share of output token. Only applies when the output token exists (note this is a ratio and not a percentage value, i.e. 1.05 instead of 105%) "
   exchangeRate: BigDecimal
```

Comment:
Again, if we are to supply multiple outputToken, then exchangeRate will need to be an array too.

Reply:
same